docusaurus.config.js

```diff
@@ -94,7 +94,8 @@ module.exports = {
       ],
       copyright: `Copyright © ${new Date().getFullYear()} PEcAn Project.`,
     },
-    hideableSidebar: true,
+
+    docs: { sidebar: { hideable: true } },
   },
```

Comment:
Please add a comment to explain this line.

Reply:
<img width="1181" alt="Screenshot 2023-03-08 at 12 30 47 PM" src="https://user-images.githubusercontent.com/96714915/223641849-199a54d5-5756-4c40-9cbc-88ab4cbf0160.png">
When I was trying to set up the project locally, I got this error so I changed the config and it starts working fine.

Reply:
<img width="1181" alt="Screenshot 2023-03-08 at 12 31 16 PM" src="https://user-images.githubusercontent.com/96714915/223642284-ea784eba-e6f0-4789-88bd-f1247988aacf.png">
But when I made PR this showed this error. How can I fix this?

Reply:
it also works when I comment out line 97